cvmfs/swissknife_sign.cc

```diff
@@ -141,9 +141,12 @@ int swissknife::CommandSign::Main(const swissknife::ArgumentList &args) {
       spooler->RegisterListener(&CommandSign::CertificateUploadCallback, this);
 
     // Safe certificate (and wait for the upload through a Future)
+    spooler->ProcessCertificate(certificate, "");
     std::string alt_path = manifest->alt_catalog_path();
-    if (alt_path.size()) {alt_path += "cert";}
-    spooler->ProcessCertificate(certificate, alt_path);
+    if (alt_path.size()) {
+      alt_path += "cert";
+      spooler->ProcessCertificate(certificate, alt_path);
```

Comment:
Should be fine, `ProcessCertificate` is a convenience wrapper for `ProcessFile` that adds the 'X' suffix to the file name.